ecommerce/frontend/src/components/SignUpForm.js

```diff
@@ -0,0 +1,311 @@
+                        required: true,
+                        message: 'Please input your E-mail!',
+                    },
+                ]}
+            >
+                <Input />
+            </Form.Item> */}
+
+            {/* Password input */}
+            {/* <Form.Item
+                name="password"
+                label="Password"
+                rules={[
+                    {
+                        required: true,
+                        message: 'Please input your password!',
+                    },
+                ]}
+                hasFeedback
+            >
+                <Input.Password />
+            </Form.Item> */}
+            
+            {/* Password confirmation input */}
+            {/* <Form.Item
+                name="confirm"
+                label="Confirm Password"
+                dependencies={['password']}
+                hasFeedback
+                rules={[
+                    {
+                        required: true,
+                        message: 'Please confirm your password!',
+                    },
+                    ({ getFieldValue }) => ({
+                        validator(rule, value) {
+                            if (!value || getFieldValue('password') === value) {
+                                return Promise.resolve();
+                            }
+
+                            return Promise.reject('The two passwords that you entered do not match!');
+                        },
+                    }),
+                ]}
+            >
+                <Input.Password />
+            </Form.Item> */}
+
+            {/* City input */}
+            {/* <Form.Item
+                id="cityInput"
+                hidden={!addressEnabled}
+                name={['address', 'city']}
+                label="City"
+                rules={ addressEnabled ?  
+                [
+                    {
+                        required: true,
+                        message: 'Please select your city!',
+                    },
+                ] : null}
+            >
+                <Input />
+            </Form.Item> */}
+            
+            {/* State input */}
+            {/* <Form.Item
+                id="stateInput"
+                hidden={!addressEnabled}
+                name={['address', 'state']}
+                label="State"
+                rules={ addressEnabled ?
+                [
+                    {
+                        required: true,
+                        message: 'Please select your state!',
+                    },
+                ] : null}
+            >
+                <Input />
+            </Form.Item> */}
+
+            {/* Zip code input */} 
+            <Form.Item
+                id="zipCodeInput"
+                // hidden={!addressEnabled}
+                name={['address', 'zipCode']}
+                label="Zip Code"
+                rules={[
+                    {
+                        required: addressEnabled,
```

Comment:
ZipCode Rules: Is there a rule for 'only numeric' input? 
Phone number Rules: Is there a rule for 'certain lengths'? 

Maybe we can improve the form answers.

Reply:
Mehdi and I worked on that today, Mehdi pushed a corrected version. So they will be controlled.

Reply:
I think for your previous comment(about the page is being messy and footer floating on top of another): it feels like it is related to the layout. I believe we should tackle that problem when putting everything together.